<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMapProperties</class>
 <widget class="QDialog" name="DialogMapProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map properties...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="WidgetTextLang" name="widgetLangName" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>408</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tileset:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxTileset">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>68</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Sizes</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="horizontalSpacing">
               <number>25</number>
              </property>
              <property name="verticalSpacing">
               <number>15</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Length:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxLength">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>16</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Width:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxWidth">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>16</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="2" rowspan="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>78</width>
                  <height>98</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Height:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxHeight">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Depth:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBoxDepth">
                  <property name="minimumSize">
                   <size>
                    <width>75</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>75</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Music:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QWidget" name="widgetChooseMusic" native="true">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="3">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>88</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Background sound:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QWidget" name="widgetChooseBackgroundSound" native="true">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Camera properties:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBoxCameraProperties">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Sky</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButtonColor">
                <property name="text">
                 <string>Color</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxColor">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="2" rowspan="2">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>38</width>
                  <height>48</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radioButtonSkybox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Skybox</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QWidget" name="widgetChooseSkybox" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Battle map:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxBattleMap">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Random battles</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
              <item>
               <widget class="QTreeView" name="treeViewRandomBattles"/>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Number of steps after next battle:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="4">
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>18</width>
                    <height>118</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="PanelPrimitiveValue" name="widgetConstantVariableSteps" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>with variation (in %):</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="PanelPrimitiveValue" name="widgetConstantVariableStepsVariation" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTextLang</class>
   <extends>QWidget</extends>
   <header>widgettextlang.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMapProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMapProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
